packages/vscode-extension/src/commands/CreateProject.ts

```diff
@@ -0,0 +1,229 @@
+/*
+ ** Copyright (c) 2021 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+
+import { Uri, commands, window } from 'vscode';
+import { ANSWERS, CREATE_PROJECT } from '../service/TranslationKeys';
+import { ApplicationConstants, FileSystemService, InteractiveAnswersValidator } from '../util/ExtensionUtil';
+import BaseAction from './BaseAction';
+import * as os from 'os';
+import * as path from 'path';
+
+const COMMAND_NAME = 'createproject';
+const COMMAND_ARGUMENTS = {
+    INCLUDE_UNIT_TESTING: 'includeunittesting',
+    OVERWRITE: 'overwrite',
+    PARENT_DIRECTORY: 'parentdirectory',
+    PROJECT_ID: 'projectid',
+    PROJECT_FOLDER_NAME: 'projectfoldername',
+    PROJECT_NAME: 'projectname',
+    PROJECT_VERSION: 'projectVersion',
+    PUBLISHER_ID: 'publisherid',
+    TYPE: 'type',
+};
+
+export default class CreateProject extends BaseAction {
+
+	constructor() {
+		super(COMMAND_NAME);
+	}
+
+    protected async execute(): Promise<void> {
+        const commandArgs = await this.getCommandArgs();
+        if (Object.keys(commandArgs).length === 0) {
+            return;
+        }
+
+        const actionResult = await this.runSuiteCloudCommand(commandArgs);
+        if (actionResult.isSuccess()) {
+            await CreateProject.openProjectInNewWindow(actionResult.projectDirectory);
```

Comment:
The story says that there should be a question asking about atttaching the project to the current window or open a new one, but I think is not implemented.

Reply:
You are correct there. I had to clarify first with the team before I implemented it. I will be adding that logic here and I'll let you know when I've added it.

Reply:
Updated to include the logic I mentioned.